ProcessHacker/proctree.c

```diff
@@ -3276,7 +3276,7 @@ BOOLEAN NTAPI PhpProcessTreeNewCallback(
                             PhInitializeStringRef(&getCellText->Text, L"x64");
                             break;
                         }
-                    case IMAGE_FILE_MACHINE_ARM:
+                    case IMAGE_FILE_MACHINE_ARMNT:
```

Comment:
probably just add `IMAGE_FILE_MACHINE_ARMNT`:

```cpp
case IMAGE_FILE_MACHINE_ARM:
case IMAGE_FILE_MACHINE_ARMNT:
```

Reply:
Would not that be incorrect? According to the [docs][1],

> All code executed on this platform is expected to start and always remain in Thumb mode


[1]: https://docs.microsoft.com/en-us/cpp/build/overview-of-arm-abi-conventions

Reply:
Yeah you're right, this is fine. Thank for fixing it.